data/crac-io/crac-io-json/src/test/resources/retrocompatibility/v1/crac-v1.8.json

```diff
@@ -0,0 +1,411 @@
+      "max" : 7,
+      "rangeType" : "absolute"
+    }, {
+      "min" : -3,
+      "max" : 3,
+      "rangeType" : "relativeToInitialNetwork"
+    } ]
+  },{
+    "id" : "pstRange4Id",
+    "name" : "pstRange4Name",
+    "operator" : "RTE",
+    "onVoltageConstraintUsageRules" : [ {
+      "instant" : "curative",
+      "voltageCnecId" : "voltageCnecId"
+    } ],
+    "networkElementId" : "pst3",
+    "groupId" : "group-3-pst",
+    "initialTap" : 1,
+    "tapToAngleConversionMap" : {
+      "-1" : 1.0,
+      "0" : 1.5,
+      "1" : 2.0,
+      "2" : 2.5,
+      "3" : 3.0,
+      "-3" : 0.0,
+      "-2" : 0.5
+    },
+    "ranges" : [ {
+      "min" : 1,
+      "max" : 7,
+      "rangeType" : "absolute"
+    }, {
+      "min" : -3,
+      "max" : 3,
+      "rangeType" : "relativeToInitialNetwork"
+    } ]
+  } ],
+  "hvdcRangeActions" : [ {
+    "id" : "hvdcRange1Id",
+    "name" : "hvdcRange1Name",
+    "operator" : "RTE",
+    "speed" : 20,
+    "onInstantUsageRules" : [ {
+      "instant" : "preventive",
+      "usageMethod" : "available"
+    } ],
+    "networkElementId" : "hvdc",
+    "initialSetpoint" : 100,
+    "ranges" : [ {
+      "min" : -1000.0,
+      "max" : 1000.0
+    } ]
+  }, {
+    "id" : "hvdcRange2Id",
+    "name" : "hvdcRange2Name",
+    "operator" : "RTE",
+    "onFlowConstraintUsageRules" : [ {
+      "instant" : "preventive",
+      "flowCnecId" : "cnec3curId"
+    } ],
+    "networkElementId" : "hvdc2",
+    "groupId" : "group-1-hvdc",
+    "initialSetpoint" : -100,
+    "ranges" : [ {
+      "min" : -1000.0,
+      "max" : 1000.0
+    } ]
+  } ],
+  "injectionRangeActions" : [ {
+    "id" : "injectionRange1Id",
+    "name" : "injectionRange1Name",
+    "operator" : null,
+    "speed" : 30,
+    "onContingencyStateUsageRules" : [ {
+      "instant" : "curative",
+      "contingencyId" : "contingency1Id",
+      "usageMethod" : "available"
+    } ],
+    "networkElementIdsAndKeys" : {
+      "generator1Id" : 1.0,
+      "generator2Id" : -1.0
+    },
+    "initialSetpoint" : 50,
+    "ranges" : [ {
+      "min" : -500.0,
+      "max" : 500.0
+    }, {
+      "min" : -1000.0,
+      "max" : 1000.0
+    } ]
+  } ],
+  "networkActions" : [ {
+    "id" : "complexNetworkActionId",
+    "name" : "complexNetworkActionName",
+    "operator" : "RTE",
+    "speed" : 40,
+    "onInstantUsageRules" : [ {
+      "instant" : "preventive",
+      "usageMethod" : "available"
+    } ],
+    "topologicalActions" : [ {
+      "networkElementId" : "ne1Id",
+      "actionType" : "close"
+    } ],
+    "pstSetpoints" : [ {
+      "networkElementId" : "pst",
+      "setpoint" : 5
+    } ]
+  }, {
+    "id" : "injectionSetpointRaId",
+    "name" : "injectionSetpointRaName",
+    "operator" : "RTE",
+    "onFlowConstraintUsageRules" : [ {
+      "instant" : "auto",
+      "flowCnecId" : "cnec3autoId"
+    } ],
+    "injectionSetpoints" : [ {
+      "networkElementId" : "injection",
+      "setpoint" : 260.0,
+      "unit" : "section_count"
```

Comment:
also add another one with a MW unit

Reply:
I couldn't add another because the version 1.0 has a hardcoded check for only one elementary action in this network action, so I replaced the section_count by megawatt